Assets/Scripts/Scenes/InGame/Enemy/Enum/EnumEnemyState.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Scenes.Ingame.Enemy
+{
+    public enum EnemyState
+    {
+        /// <summary>
+        /// �Ȃɂ��o�^����Ă��Ȃ����
+        /// </summary>
+        None,
+        /// <summary>
+        /// ����
+        /// </summary>
+        Patorolling,
+        /// <summary>
+        /// ���G
+        /// </summary>
+        Searching,
+        /// <summary>
+        /// �ǐ�
+        /// </summary>
+        Chese,
+        /// <summary>
+        /// �U��
+        /// </summary>
+        Atack,

```

Comment:
攻撃の綴りが違います。細かいですが変更をお願いします。
EnemyStatusの攻撃の綴りも間違っているのでそちらもお願いします。
```suggestion
        Attack,
```